src/features/game/lib/gameMachine.ts

```diff
@@ -246,7 +246,7 @@ export type UpdateUsernameEvent = {
 
 type PostEffectEvent = {
   type: "POST_EFFECT";
-  effect: Effect;
+  effect: any;
```

Comment:
👀 Is this intentional?

Reply:
It was intentional because the type `Effect` was not actually correct. It was a single type `marketplace.listed` but was being used by other marketplace `events` also. 

We can set up more of a generic union type. I can do it in the purchase PR.